### PR TITLE
Add screenshot to readme

### PR DESCRIPTION
**Pull request by rbren**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,10 @@
 
 By using the kubernetes [vertical-pod-autoscaler](https://github.com/kubernetes/autoscaler/tree/master/vertical-pod-autoscaler) in recommendation mode, we can see a suggestion for resource requests on each of our apps. This tool creates a VPA for each deployment in a namespace and then queries them for information.
 
+<div align="center">
+<img src="img/screenshot.png" alt="Goldilocks Screenshot" />
+<br>
+
```
Comment (mgoodhart5) [inline README.md]: I just viewed this file and it looks like this addition made everything below it also be center aligned. Does the <div> need to be closed?

Comment (rbren) [reply README.md]: Whoops good catch! Thanks.

Here's what the end result looks like: https://github.com/FairwindsOps/goldilocks/tree/rb/screenshot

Comment (mgoodhart5) [reply README.md]: Yay! It looks great!! ✨

File: README.md [hunk 2]
```diff
@@ -11,6 +11,10 @@
 
 By using the kubernetes [vertical-pod-autoscaler](https://github.com/kubernetes/autoscaler/tree/master/vertical-pod-autoscaler) in recommendation mode, we can see a suggestion for resource requests on each of our apps. This tool creates a VPA for each deployment in a namespace and then queries them for information.
 
+<div align="center">
```
Comment (sudermanjr) [inline README.md]: Can you put some text above this to explain what we are seeing here?  I don't think it's clear that this is the Goldilocks dashboard.  Perhaps it should be placed in a different spot?

Comment (rbren) [reply README.md]: Good point!

**Pull request merged**